nilearn/version.py

```diff
@@ -56,17 +72,16 @@ def _import_module_with_version_check(
     """
     from distutils.version import LooseVersion
 
-    try:
-        module = __import__(module_name)
-    except ImportError as exc:
+    module, module_status = get_module_status(module_name)
+    if module_status['installed'] is False:
         user_friendly_info = ('Module "{0}" could not be found. {1}').format(
             module_name,
             install_info or 'Please install it properly to use nilearn.')
-        exc.args += (user_friendly_info,)
-        raise
+        raise ImportError(user_friendly_info)
```

Comment:
I had a look the other day and I would do something like this instead:

```py
exc.args += (user_friendly_info,)
# Necessary for python 3
if hasattr(exc, 'msg'):
    exc.msg += '. ' + user_friendly_info
```

This is because the repr/str of ImportError objects was changed in Python 3 ...